testutils/api/inventory_v2.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Northern.tech AS
```

Comment:
will something not complain about that year?

Reply:
We don't run copyright checks in tests.

Reply:
but still that is not a true year of Copyright ;-)

Reply:
no thing complained... but fixed regardless, thanks